test/nieSyntaxSpecs.js

```diff
@@ -0,0 +1,42 @@
+describe('Testing nie syntax services', function() {
+	var $injector;
```

Comment:
You don't need this member variable because you won't use the injector inside the tests.